interactions/client/bot.py

```diff
@@ -1416,6 +1421,33 @@ async def modify(
 
         return User(**data)
 
+    async def get(self, obj: Type[_T], **kwargs) -> _T:
+        """
+        A helper method for retrieving data from the Discord API in its object representation.
+
+        :param obj: The object to get. Should be a class object (not an instance!). For example: `interactions.Channel`.
+        :type obj: object
+        :param **kwargs: The arguments to pass to the HTTP method.
+        :type **kwargs: dict
+        :return: The object we're trying to get.
+        :rtype: object
+        """
```

Comment:
```suggestion
        r"""
        A helper method for retrieving data from the Discord API in its object representation.

        :param obj: The object to get. Should be a class object (not an instance!). For example: `interactions.Channel`.
        :type obj: object
        :param \**kwargs: The arguments to pass to the HTTP method.
        :type \**kwargs: dict
        :return: The object we're trying to get.
        :rtype: object
        """
```

Reply:
Flake 8 complained with the `\` before tho

Reply:
That's how you do it for the docstring if you have **kwargs, it's also elsewhere, you can check the others out

Reply:
I'll merge it and see if Flake8 complains again